Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "lwa_fm"
 edition = "2021"
 homepage = "https://github.com/Leinnan/lwa_fm"
+build = "build.rs"
```

Comment:
Okay, now I get it. If you create `build.rs` it will be used and there is no need to specify it in `Cargo.toml`. This field is when you want to use different path than `REPO_ROOT/build.rs`. So setting up it to default value add no value.